src/app/containers/MediaPlayer/index.jsx

```diff
@@ -50,32 +58,38 @@ const MediaPlayerContainer = ({ blocks, placeholder }) => {
 
   const placeholderSrc = getPlaceholderSrc(imageUrl);
   const embedSource = embedUrl({
-    requestUrl: `${id}/${versionId}/${lang}`,
-    type: 'articles',
+    requestUrl: `${assetId}/${versionId}/${lang}`,
+    type,
     isAmp,
     origin,
   });
 
   return (
-    <GridItemConstrainedMedium>
+    <Wrapper>
       <Metadata aresMediaBlock={aresMediaBlock} />
       {isAmp ? (
-        <AmpMediaPlayer src={embedSource} placeholderSrc={placeholderSrc} />
+        <AmpMediaPlayer
+          src={embedSource}
+          showPlaceholder={showPlaceholder}
+          placeholderSrc={placeholderSrc}
+        />
       ) : (
         <CanonicalMediaPlayer
           src={embedSource}
+          showPlaceholder={showPlaceholder}
           placeholder={placeholder}
```

Comment:
Hey @dr3, have you noticed `placeholder` is never actually used? Somewhere down the line is appears it has been confused with `showPlaceholder`.